### DISCUSSION TITLE
Where pdns refers for Root domain?

### DISCUSSION THREAD

**Discussion by muzuco** [PowerDNS/pdns#13482] [Q&A] [answered]
Hi, I'm using pdns auth server and recursor in same PC(CentOS 6.4).
pdns.conf is:

```
recursor=127.0.0.1:53
```

recursor.conf is:

```
local-address=127.0.0.1
forward-zones=test.com=222.XXX.XXX.XXX;222.XXX.XXX.XXX
```

So when pdns receive a DNS request about "test.com", it will hand over that to 222.XXX.XXX.XXX.

If pdns receive a DNS request not about "test.com"(even also pdns auth server doesn't have any records for this request), how could it resolve?
pdns will refer a "/etc/resolv.conf" to know Root domain's IP? 
I'm wondering how pdns can resolve the other DNS requests even if I didn't set anything except pdns.conf file and recursor.conf file.

**Comment by Habbie** [comment]
Hello,

our github issue tracker is not a support forum. Please see https://www.powerdns.com/opensource.html for information about our IRC channel and mailing lists, where we will be happy to help you.

Closing ticket.

**Comment by llorllale** [comment]
I've been hunting down the answer to this question for the past hour or so; really unfortunate no answer was provided here

**Comment by mnordhoff** [comment, marked as answer]
https://doc.powerdns.com/recursor/appendices/FAQ.html#handling-of-root-hints

The GitHub issue tracker is still not a support forum, though GitHub discussions are.

Edit:

The key term is the "root hints", a list of the root nameserver names and IP addresses distributed in a [file](https://www.iana.org/domains/root/files) by IANA. Recursive resolvers usually include a copy -- hardcoded in the binary, as a separate configuration file, or both. They use the root hints to bootstrap, as described at the link above.
